src/init.ts

```diff
@@ -215,5 +215,9 @@ export async function init(options: Options): Promise<boolean> {
     options.logger.log('No edits needed in package.json.');
   }
   await generateTsConfig(options);
+
+  // Run `npm install` after initial setup so `npm run check` works right away
+  cp.spawnSync('npm', ['install'], {stdio: 'inherit'});
```

Comment:
Question:
* Should this happen if we didn't actually modify dependencies? My guess is yes.
* Should this happen if the user passed `--dry-run`? My guess is no.
* Can you add a test to `test-kitchen`?

Reply:
Answers!
- Yes, I think so.
- Nope, fixed :)
- Done.  Was more of an adventure than I expected.